egs/tedlium/s5/local/nnet3/run_tdnn_cleanup.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
+
```

Comment:
get rid of this script too.  You can copy it to run_tdnn.sh if it's reasonably back-compatible (small configuration changes may be OK for all these scripts as long as they don't hurt results).